### ISSUE TITLE
Add call start time to Call info

### ISSUE THREAD

**Issue by ToniaDemchuk** [Azure/communication-ui-library#3083]
**What feature are you requesting?**
API lacks the call start time information. Some `callStartTime` exists as private prop, but it is fully dependent on when the current instance of `Call` is created after `callAdapter.joinCall(...)`.
In case of one user reconnects, there is no possibility to know where the call actually started.

**What would be the benefit of adding this feature?**
To implement additional functionality of timers during the call.
To know exact duration of a call.

**What solution would be ideal for you?**
`CallInfo` object has callStartTime property.

**What alternatives have you considered?**
SDK apis to get call information including call start time, duration, is call in progress.

**Is there any additional information for your request?**

**Attach screenshots or images add detail to your request:**

**Comment by carocao-msft** [comment]
Hi, Thanks for reaching out to us, we are currently looking into this. Will update you asap :)

**Comment by akania** [comment]
HI @ToniaDemchuk - thank you for suggestion - we'll put this on the roadmap, we don't plan to add it as of today, 
To unblock you - the way to implement this on application side would be to subscribe to call state and start counting time from the moment call goes to "Connected" state and finish it when it goes to "Disconnected" 
Time should be stopped when user goes on hold/resumes the call
a point about reconnecting is valid - though we wouldn't stop timer in such case.
Let me know if this will work for you

**Comment by ToniaDemchuk** [comment]
Thanks for response.
For now, suggested workaround can work for me.

Also it can be good to know some statistics about the call in general - the whole group call start/end time, number of total participants during the whole call, etc.

**Comment by dmceachernmsft** [comment]
Closing issue due to inactivity. if there is anything else you need please feel free to open the issue again! 😊
